### ISSUE TITLE
Failed to run optimizer ArithmeticOptimizer

### ISSUE THREAD

**Issue by sed0724963** [tensorflow/models#6215]
I had upgrade my tensorflow object detection API, because I want to plot the validation loss on tensorboard,In the Old version is only plot the total loss for training .

and I'm use the  macos system , tensorflow is cpu version.
the different on new and old version is  : train.py and model_main.py.

now, I am use the model_main.py ,but it's have a error,anyone knows it?
<img width="759" alt="2019-02-17 7 36 38" src="https://user-images.githubusercontent.com/38509051/52912358-641f7f80-32eb-11e9-8a14-7389c6040809.png">

------------------------

### System information
- **What is the top-level directory of the model you are using**:
./models/research
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
NO
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
MacOS High Sierra
- **TensorFlow installed from (source or binary)**:
pip3
- **TensorFlow version (use command below)**:
1.13.0-rc2
- **Bazel version (if compiling from source)**:
don't know
- **CUDA/cuDNN version**:
no
- **GPU model and memory**:
cpu only
- **Exact command to reproduce**:
don't know

**Comment by Shashi2411** [comment]
I'm experiencing the same issue.
Have you solved it?

**Comment by sed0724963** [comment]
@Shashi2411 Not yet

**Comment by v-qjqs** [comment]
I'm experiencing the same issue on win notebook with py3.7, tf-gpu1.13, cuda10

**Comment by abh2424** [comment]
I am still facing this issue. Please help me to solve the error.

**Comment by mttbx** [comment]
Me too. It seems the preprocessing problem, I can still train my network although.

**Comment by Tdoe4321** [comment]
I as well am having this issue with a faster_rcnn network, anyone have any ideas about it yet?

**Comment by mehtanaman-awl** [comment]
I have the issue, but my Memory usage and Swp are used 100%. And the system hangs(I'm guessing it may as well be due to some other reason, but not sure why the system hangs, kswapd0 consumes 50% of memory).
Ubuntu 18.04, 16GB RAM, tf-gpu 13.1.0, cuda 10, python 3.6
